src/aragonjs-wrapper.js

```diff
@@ -240,7 +239,7 @@ const subscribe = (
         )
       )
     }),
-    permissions: permissions.subscribe(throttle(onPermissions, 100)),
+    permissions: permissions.subscribe(onPermissions),
```

Comment:
@bpierre We've since added a 30ms debounce to the permissions observable, so I've removed this.